### PR TITLE
[res] Update circle schema comment

### PR DESCRIPTION
**Pull request by jinevening**
This adds comment for min/max values of QuantizationParameters.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

### CODE REVIEW

**Comment by jinevening** [comment]
For reviewers:

This change is requested by @parjong to clarify (leave a document) the role of `min/max` fields.

**Review by glistening — APPROVED** [review]
LGTM

**Comment by jinevening** [comment]
I've added @glistening and @hseok-oh as reviewers, because I'll update circle schema for runtime after this PR.

**Comment by jinevening** [comment]
I've applied suggestions from messenger discussion.

I copy & pasted messenger conversation after removing sensitive contents.

```
[박종현] 2023-10-31 08:47
INVARIANT 1. scale.size == zero_point.size 

INVARIANT 2.
- min.size == 0 && max.size == 0, or
- min.size == max.size == scale.size

[박종현] 2023-10-31 08:48
말씀하신 내용을 듣고 대충 생각해보면 이런식이지 않을까 싶네요

[박종현] 2023-10-31 08:48
size가 아니라 length라고 해야하려나요

[정은지] 2023-10-31 08:53
[Valid if]

* min/max 의 length 가 0인 경우  (no info)
* min/max 의 length 가 scale/zerop 의 length보다 같은 경우

[Undefined behavior]

* 나머지 (min/max 의 length 가 scale/zerop 의 length 와 다른 경우)

[정은지] 2023-10-31 08:54
현재는 이정도로 해석하면 되는 것 같고 필요하면 circle에도 이정도 써두면 될까요 ㅎㅎ

[박종현] 2023-10-31 08:54
넵!

[정혁진 (Jin)] 2023-10-31 08:59
  min:[float];  // For importing back into tensorflow.

circle schema 에서는 여기 있는 주석을 없애는 것도 좋을 것 같네요.

[정은지] 2023-10-31 09:00
네 저희와는 크게 관련없어 보이더라고요
```

**Review by mhs4670go — APPROVED** [review]
LGTM

**Comment by jinevening** [comment]
@nnfw-bot test nncc-debug

**Review by glistening — APPROVED** [review]
LGTM

**Review by hseok-oh — APPROVED** [review]
LGTM

File: res/CircleSchema/0.6/circle_schema.fbs [hunk 1]
```diff
@@ -84,8 +84,12 @@ table QuantizationParameters {
...
   //   f = scale * (q - zero_point)
   // For other quantization types, the QuantizationDetails below is used.
-  min:[float];  // For importing back into tensorflow.
-  max:[float];  // For importing back into tensorflow.
+  // NOTE min/max values are valid if
+  // 1. length of min/max == 0 or
+  // 2. length of min/max == length of scale/zero_point
+  // Otherwise, min/max are not valid (undefined behavior).
+  min:[float];
+  max:[float];
```
Comment (seanshpark) [inline res/CircleSchema/0.6/circle_schema.fbs:92]: I'm not sure about the versioning but is it OK to go with same `0.6` with these changes?

Comment (seanshpark) [reply res/CircleSchema/0.6/circle_schema.fbs:92]: @jinevening , have you tests with files generated before-after these attributes?

Q) why not add at the end?

Comment (jinevening) [reply res/CircleSchema/0.6/circle_schema.fbs:92]: Ah, this PR just updates comments (about min/max). So I think we don't have to increase version number.

Comment (seanshpark) [reply res/CircleSchema/0.6/circle_schema.fbs:92]: Ah, I missed "before" codes ... 😭

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**